facade/host.go

```diff
@@ -118,6 +118,42 @@ func (f *Facade) UpdateHost(ctx datastore.Context, entity *host.Host) error {
 	return err
 }
 
+// RestoreHosts restores a list of hosts, typically from a backup
+func (f *Facade) RestoreHosts(ctx datastore.Context, hosts []host.Host) error {
+	var exists bool
+	var err error
+
+	for _, host := range hosts {
+		host.DatabaseVersion = 0
```

Comment:
Seems like it makes more sense to move this before line 144 so we're only changing properties of this host object if we know we're about to restore it.